specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcechanges.json

```diff
@@ -0,0 +1,405 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Resource Graph",
+    "description": "Azure Resource Graph API Reference",
+    "version": "2020-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ResourceGraph/resourceChanges": {
+      "post": {
```

Comment:
Feels this can be supported with GET particularly you have $top, $skip which is typical of query string parameters. But I will leave ARM reviewer to chime in here.

Reply:
Unfortunately no because GET doesn't allow a request body. (There's a "body" parameter below.)

---
In reply to: [508220282](https://github.com/Azure/azure-rest-api-specs/pull/11070#discussion_r508220282) [](ancestors = 508220282)

Reply:
Well technically speaking some of the parameters in the request body could probably be moved to query string, but I'm not going to insist on that given that the rest of the RP is modeled using POST.

---
In reply to: [552319239](https://github.com/Azure/azure-rest-api-specs/pull/11070#discussion_r552319239) [](ancestors = 552319239,508220282)